docs/configuring-playbook-ntfy.md

```diff
@@ -14,9 +14,15 @@ The playbook can install and configure the [ntfy](https://ntfy.sh/) push notific
 
 Using the [UnifiedPush](https://unifiedpush.org) standard, ntfy enables self-hosted (Google-free) push notifications from Matrix (and other) servers to UnifiedPush-compatible Matrix compatible client apps running on Android and other devices.
 
-This role is intended to support UnifiedPush notifications for use with the Matrix and Matrix-related services that this playbook installs. This role is not intended to support all of ntfy's other features.
+See the project's [documentation](https://docs.ntfy.sh/) to learn what it does and why it might be useful to you.
 
-**Note**: In contrast to push notifications using Google's FCM or Apple's APNs, the use of UnifiedPush allows each end-user to choose the push notification server that they prefer. As a consequence, deploying this ntfy server does not by itself ensure any particular user or device or client app will use it.
+**Notes**:
+- To make use of your ntfy installation, you need installing `ntfy` app on your device and configuring your UnifiedPush-compatible Matrix client. **Otherwise your device will not receive push notifications from the ntfy server.** Refer [this section](#usage) for details.
+- This playbook focuses on setting up a ntfy server for getting it send UnifiedPush notifications to Matrix-related services that this playbook installs, while the installed server will be available for other non-Matrix apps as well like [Tusky](https://tusky.app/) and [DAVx⁵](https://www.davx5.com/). This playbook does not intend to support all of ntfy's features.
+
+The Ansible role for ntfy is developed and maintained by [the MASH (mother-of-all-self-hosting) project](https://github.com/mother-of-all-self-hosting/ansible-role-ntfy). For details about configuring ntfy, you can check them via:
```

Comment:
Here, the link is named **the MASH (mother-of-all-self-hosting) project**.

Seeing this, I (and possibly others) would probably expect it to link to the MASH project - most likely to the Github organization: https://github.com/mother-of-all-self-hosting

Linking to to the `ansible-role-ntfy` repository is surprising.

Perhaps something like this is better:

```suggestion
The [Ansible role for ntfy](https://github.com/mother-of-all-self-hosting/ansible-role-ntfy) is developed and maintained by [the MASH (mother-of-all-self-hosting) project](https://github.com/mother-of-all-self-hosting). For details about configuring ntfy, you can check them via:
```

Reply:
Addressed with 6445051.